packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -652,33 +651,24 @@ contract PricelessPositionManager is FeePayer {
      * @notice Accessor method for a sponsor's collateral.
      * @dev This is necessary because the struct returned by the positions() method shows
      * rawCollateral, which isn't a user-readable value.
-     * @dev TODO: This method does not account for any pending regular fees that have not yet been withdrawn
-     * from this contract, for example if the `lastPaymentTime != currentTime`. Future work should be to add
-     * logic to this method to account for any such pending fees.
+     * @dev This method accounts for pending regular fees that have not yet been withdrawn from this contract, for
+     * example if the `lastPaymentTime != currentTime`.
      * @param sponsor address whose collateral amount is retrieved.
      * @return collateralAmount amount of collateral within a sponsors position.
      */
-    function getCollateral(address sponsor)
-        external
-        view
-        nonReentrantView()
-        returns (FixedPoint.Unsigned memory collateralAmount)
-    {
+    function getCollateral(address sponsor) external view nonReentrantView() returns (FixedPoint.Unsigned memory) {
         // Note: do a direct access to avoid the validity check.
-        return _getFeeAdjustedCollateral(positions[sponsor].rawCollateral);
+        return _getPendingRegularFeeAdjustedCollateral(_getFeeAdjustedCollateral(positions[sponsor].rawCollateral));
```

Comment:
here the internal `_getPendingRegularFeeAdjustedCollateral` method is applied. Note that it NOT applied anywhere else as all other functions include the `fees()` modifier which means that pending regular fees are already accounted for.